Kickstarter-iOS/Assets.xcassets/icons/verified-checkmark-icon.imageset/Contents.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "Mask.pdf"
```

Comment:
The border seems a bit thicker than the designs, perhaps we could export the checkmark without the border and apply that programmatically?

| Designs  | Implementation |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/3735375/75723994-44f35600-5c92-11ea-9c23-fd0c7bee620e.png) | ![image](https://user-images.githubusercontent.com/3735375/75724176-aadfdd80-5c92-11ea-8901-ee883280e877.png) |